source/features/download-folder-button.tsx

```diff
@@ -1,25 +1,34 @@
+import './download-folder-button.css';
 import * as pageDetect from 'github-url-detection';
+import {$} from 'select-dom';
 
 import features from '../feature-manager.js';
 import observe from '../helpers/selector-observer.js';
 
-function add({parentElement: deleteDirectoryItem}: HTMLAnchorElement): void {
-	const item = deleteDirectoryItem!.cloneNode(true);
+function add(container: HTMLElement): void {
+	const dropdownItem = $('[aria-keyshortcuts="c"]:first-child', container);
+
+	if (!dropdownItem)
+		return;
+
+	const item = dropdownItem!.cloneNode(true);
+	item.replaceChildren(document.createElement('a'));
```

Comment:
I think you're missing the classes, we shouldn't need custom CSS to match the color. 

Does the "copy path" button classes already handle the `a` style?

Reply:
No it doesn't.